tests/integration/ceph_flex_test.go

```diff
@@ -87,9 +87,9 @@ func (s *CephFlexDriverSuite) SetupSuite() {
 	flexTestCluster := TestCluster{
 		namespace:               s.namespace,
 		storeType:               "bluestore",
-		storageClassName:        "",
+		storageClassName:        "gp2",
```

Comment:
We can't default to `gp2`, this should be controlled by an env var

Reply:
Done.